index.html

```diff
@@ -5,6 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Soul Shoes</title>
+    <style>
+        body {
+            background-color: red;
```

Comment:
I'm not sure that this color theme will accessible to all users.

Reply:
Oh! That's a great point! I forgot about red-green color-blindness. You're so smart.
I'll fix that now!